components/colorarea/metadata/colorarea.yml

```diff
@@ -32,8 +32,8 @@ examples:
           </svg>
         </div>
 
-        <input type="range" class="spectrum-ColorArea-slider" name="x" aria-label="saturation and value" min="0" max="1" step="0.01">
-        <input type="range" class="spectrum-ColorArea-slider" name="y" aria-label="saturation and value" min="0" max="1" step="0.01">
+        <input type="range" class="spectrum-ColorArea-slider" name="x" aria-label="saturation and value" min="0" max="1" step="0.01" tabindex="-1">
+        <input type="range" class="spectrum-ColorArea-slider" name="y" aria-label="saturation and value" min="0" max="1" step="0.01" tabindex="-1">
```

Comment:
I'm assuming you have a reason for removing these inputs from the tab order and adding in the div for the handle instead? I know I can't currently drag it so it's hard to test but that of course removes those from being accessible

Reply:
Yes! Previously the ColorHandles did not receive any visible keyboard focus. I added a tabIndex of 0 to the ColorHandle to add it to Tab order so we can now see the keyboard focus styling . I removed the slider inputs from the tab order because they are hidden from view and were causing a pause or skipping when tabbing through the component. 

[Also a pattern SWC has in place too](https://github.com/adobe/spectrum-web-components/blob/d4e49f581d41720d92e9f786cda4af56dbafc8c5/packages/color-area/src/ColorArea.ts#L423)

I am curious if @pfulton has any concerns about this that I might now be aware of though.

Reply:
Ok what SWC has in place functions with a keyboard and with a screen reader, that was my main concern. The only thing is that they have one input labeled for saturation and one for luminosity. Should we keep that consistent here to differentiate which value the user is changing?